resources/oe/src/oeCore.i

```diff
@@ -190,6 +197,21 @@ PROCEDURE LOCAL_GET_TABLE_DETAILS:
         DELETE OBJECT qhField.
         DELETE OBJECT bhFile.
 
+        DO i = 1 TO 2:
```

Comment:
Can you clarify why is it from 1 to 2? And why the iFieldOrder is increased by 10?

Reply:
Do loop is for creating ROWID and RECID. Instead of two columns of assign statements this DO loop was used. iFieldOrder increaced by 10 also for ROWID and RECID creation - since they are last columns of table, so iFieldOrder also should be last.